src/Frontend/Page/Delegation.hs

```diff
@@ -14,31 +14,89 @@ where
 import Prelude
 
 import Access
+import Control.Arrow ((&&&))
+
+import Action (ActionM, currentUser, delegateTo, equery)
 import Frontend.Core
 import Frontend.Prelude
+import Persistent
 
 import qualified Frontend.Path as U
-
+import qualified Text.Digestive.Form as DF
+import qualified Text.Digestive.Lucid.Html5 as DF
 
 -- | 12. Delegate vote
-data PageDelegateVote = PageDelegateVote
+data PageDelegateVote = PageDelegateVote DScopeFull [User]
   deriving (Eq, Show, Read)
 
-instance ToHtml PageDelegateVote where  -- FIXME: remove bogus instance.
-    toHtmlRaw = toHtml
-    toHtml p = semanticDiv p "PageDelegateVote"
-
 instance Page PageDelegateVote where isAuthorized = adminPage -- FIXME who needs to see this
 
-instance FormPage PageDelegateVote where  -- FIXME
-    type FormPagePayload PageDelegateVote = ()
-    formAction _   = U.Broken
-    redirectOf _ _ = U.Broken
-    makeForm _     = pure ()
-    formPage _ _ _ = pure ()
+newtype PageDelegationVotePayload = PageDelegationVotePayload
+    { unPageDelegationVotePayload :: AUID User }
+  deriving (Eq, Show, Read)
+
+instance FormPage PageDelegateVote where
+    type FormPagePayload PageDelegateVote = PageDelegationVotePayload
+
+    formAction (PageDelegateVote scope _users) = case scope of
+        DScopeGlobalFull          -> U.Broken
```

Comment:
No delegation page is rendered in this situation. Should we remove this explicitly? @fisx